tools/pkglint/package.json

```diff
@@ -18,8 +18,8 @@
   "scripts": {
     "build": "tsc -b && eslint . --ext=.ts && chmod +x bin/pkglint",
     "test": "echo success",
-    "build+test": "npm run build && npm test",
-    "build+test+package": "npm run build+test",
+    "build+test": "yarn run build && yarn test",
+    "build+test+package": "yarn run build+test",
```

Comment:
Is any reason for these changes?

Reply:
This was done by CDK as well: https://github.com/aws/aws-rfdk/pull/297